src/preamble_minimal.js

```diff
@@ -56,26 +65,25 @@ var HEAP8, HEAP16, HEAP32, HEAPU8, HEAPU16, HEAPU32, HEAPF32, HEAPF64,
 #endif
   wasmMemory, wasmTable;
 
-
 function updateMemoryViews() {
   var b = wasmMemory.buffer;
 #if ASSERTIONS && SHARED_MEMORY
   assert(b instanceof SharedArrayBuffer, 'requested a shared WebAssembly.Memory but the returned buffer is not a SharedArrayBuffer, indicating that while the browser has SharedArrayBuffer it does not have WebAssembly threads support - you may need to set a flag');
 #endif
 #if SUPPORT_BIG_ENDIAN
-  HEAP_DATA_VIEW = new DataView(b);
-#endif
-  HEAP8 = new Int8Array(b);
-  HEAP16 = new Int16Array(b);
-  HEAP32 = new Int32Array(b);
-  HEAPU8 = new Uint8Array(b);
-  HEAPU16 = new Uint16Array(b);
-  HEAPU32 = new Uint32Array(b);
-  HEAPF32 = new Float32Array(b);
-  HEAPF64 = new Float64Array(b);
+  {{{ maybeExport('HEAP_DATA_VIEW') }}} HEAP_DATA_VIEW = new DataView(b);
+#endif
+  {{{ maybeExport('HEAP8') }}} HEAP8 = new Int8Array(b);
+  {{{ maybeExport('HEAP16') }}} HEAP16 = new Int16Array(b);
+  {{{ maybeExport('HEAP32') }}} HEAP32 = new Int32Array(b);
+  {{{ maybeExport('HEAPU8') }}} HEAPU8 = new Uint8Array(b);
+  {{{ maybeExport('HEAPU16') }}} HEAPU16 = new Uint16Array(b);
+  {{{ maybeExport('HEAPU32') }}} HEAPU32 = new Uint32Array(b);
+  {{{ maybeExport('HEAPF32') }}} HEAPF32 = new Float32Array(b);
+  {{{ maybeExport('HEAPF64') }}} HEAPF64 = new Float64Array(b);
 #if WASM_BIGINT
-  HEAP64 = new BigInt64Array(b);
-  HEAPU64 = new BigUint64Array(b);
+  {{{ maybeExport('HEAP64') }}} HEAP64 = new BigInt64Array(b);
+  {{{ maybeExport('HEAPU64') }}} HEAPU64 = new BigUint64Array(b);
```

Comment:
Avoid adding an extra space between the template the declaration, and instead add the space after the `=` withing the template on line 11